packages/app/src/sagas/answers-references/load.js

```diff
@@ -43,23 +43,25 @@ export default function* load({ meta: { pagesIndex } }) {
     }
 
     /** @type {{ data: Answer.Reference[]; pagesLength: number; totalLength: number; }} */
-    const { data: answersReferences, pagesLength, totalLength } = yield request.get(
-      "/answers_references",
-      true,
-    );
+    // eslint-disable-next-line prettier/prettier
+    const {
+      data: answersReferences,
+      pagesLength,
+      totalLength,
+    } = yield request.get("/answers_references", true);
 
     const answerIds = answersReferences.map(({ answer_id }) => answer_id);
 
     const { data: answers } = yield customPostgrester()
       .select("id")
-      .select("agreement(idcc,name)")
-      .select("question(index, value)")
+      .select("agreement:agreements!answers_agreement_id_fkey(idcc,name)")
```

Comment:
Un peu moins clair qu'avant l'écriture :(

Reply:
Oui malheureusement il n'y a pas d'autre solution. PosgREST v7+ bugue si on ne précise pas la foreign key qui fait la relation entre les tables. Et le nom précédent les `:` sert à renommer la propriété pour garder les changements au minimum et aussi parce que le pluriel devrait être réservé aux arrays, or c'est bien un objet qu'reçoit là (one-to-many) et non une collection (many-to-many).